manifests/foreman_proxy.pp

```diff
@@ -100,14 +100,13 @@
       key_pair => Cert[$foreman_proxy_client_cert_name],
     } ->
     pubkey { $foreman_ssl_ca_cert:
-      key_pair => $::certs::server_ca,
+      key_pair => $::certs::default_ca,
     } ~>
     key_bundle { $foreman_proxy_ssl_client_bundle:
       key_pair => Cert[$foreman_proxy_client_cert_name],
     } ~>
     file { $foreman_proxy_ssl_client_bundle:
       ensure => file,
-      owner  => $::certs::group,
```

Comment:
This is a bug as well. The file is world readable, setting the group to `qpid` doesn't make any sense for a `foreman_proxy` related file.